app/app.iml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
app.iml shouldn't be committed. There was a bit of conflicted ideas earlier whether it should be committed or not. But since 1.5.1  (as I can remember) Android studio adds *.iml to .gitignore. But since our project already have the .iml file remove that too.